os/drivers/seclink/seclink_drv_utils.c

```diff
@@ -0,0 +1,46 @@
+#include <tinyara/config.h>
+#include <stdio.h>
+#include <errno.h>
+#include "seclink_drv_utils.h"
+
+#define NSPERSEC 1000000000
+#include <fcntl.h>
+#include <tinyara/timer.h>
+// microsecnd
+
+void get_frt_time(struct timer_status_s *ts)
+{
+	struct timer_status_s before;
+    struct timer_status_s after;
+    int frt_fd;
+
+    frt_fd = open("/dev/timer0", O_RDONLY);
+    if (frt_fd < 0) {
+        lldbg("ERROR: Failed to open, errno %d\n", errno);
+        return;
+    }
+
+    ioctl(frt_fd, TCIOC_SETMODE, MODE_FREERUN);
+    ioctl(frt_fd, TCIOC_START, 0);
+    ioctl(frt_fd, TCIOC_GETSTATUS, (unsigned long)(uintptr_t)ts);
```

Comment:
There is no use of this function, but I'm worried.
Maybe you call this function for measuring the elapsed time like below.
```
get_frt_time();
DO SOMETHING;
get_frt_time();
CALCULATE THE DIFF;
```
But this is not the real `DO SOMETHING` time, because it includes other functionality times like open, setmode and so on.
And in this case, we open and start the timer twice. Then it can be failed.